jkamz/Questioner
# Ft landing page 162981977

### What does this PR do?
This PR creates the layout and homepage for the site

### Description of the tasks to be completed
The tasks for this PR include:

- Create the navigation bar
- Create the sign in link
- Create links to popular meetups

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/Questioner`

`$ cd Questioner`

Check out to the PR's branch
`$ git checkout ft-landing-page-162981977`

Navigate to UI/templates folder and open index.html on browser

### What are the relevant Pivotal Tracker stories?
#162981977

### Screenshots

![homepage](https://user-images.githubusercontent.com/11965186/50722380-47ced700-10df-11e9-8c73-19010be4c7db.png)


Mobile Site

![mobile homepage](https://user-images.githubusercontent.com/11965186/50722378-371e6100-10df-11e9-80e6-4808044ca8b6.png)
